player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/templates/end.html

```diff
@@ -160,9 +160,14 @@ <h2 class="ps-nav-title">Next</h2>
                                 </a>
                                 <p class="ps-action-text">Replay</p>
                             </li>
-                            <li class="ps-action-list__item"><a class="ps-btn ps-btn--blue"><span class="user-name">{{currentUser.name.charAt(0).toUpperCase() + string.slice(1) || currentUser.handle.charAt(0).toUpperCase() + string.slice(1)}}</span></a>
```

Comment:
Do we really need to change this code?

Reply:
yes, its taking the first letter from the current user to display in icon.